protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -2677,6 +2684,46 @@ message ListLiquidityProvidersResponse {
   LiquidityProviderConnection liquidity_providers = 1;
 }
 
+// Paid liquidity fees request.
+message ListPaidLiquidityFeesRequest {
+  // Market ID to get paid liquidity fee statistics for.
+  // If omitted the statistics for all markets will be returned.
+  optional string market_id = 1;
+  // Asset ID to get paid liquidity fee statistics for.
+  // If omitted the statistics for all assets will be returned.
+  optional string asset_id = 2;
+  // Epoch to get paid liquidity fee statistics for.
+  // If omitted, the last epoch published is used.
```

Comment:
I'm guessing "published" here means the last epoch that has finished? I think I'd prefer "complete" or "finished" really. Any opinions @candida-d ?
```suggestion
  // Epoch to get paid liquidity fee statistics for. If omitted, the last epoch complete is used.
```